life_dashboard/core_stats/views.py

```
@@ -1,3 +1,4 @@
 from django.shortcuts import render
+from life_dashboard.core_stats.models import CoreStat
```

Ruff (0.11.9)

`life_dashboard.core_stats.models.CoreStat` imported but unused

Remove unused import: `life_dashboard.core_stats.models.CoreStat`

(F401)